pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -397,7 +397,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
             // At default info level, suppress all subsequent exceptions that are thrown when the connection has already
             // failed
             if (log.isDebugEnabled()) {
-                log.debug("[{}] Got exception: {}", remoteAddress, cause);
+                log.debug("[{}] Got exception: {}", remoteAddress, cause.getMessage(), cause);
```

Comment:
Is it worth using `ClientCnx.isKnownException(cause) ? cause : ExceptionUtils.getStackTrace(cause)` link we do on line 391?

Reply:
Done